src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-details/osd-details.component.spec.ts

```diff
@@ -29,7 +30,7 @@ describe('OsdDetailsComponent', () => {
       CephModule,
       CoreModule
     ],
-    declarations: [],
+    declarations: [OsdDetailsComponent, OsdPerformanceHistogramComponent, SmartListComponent],
```

Comment:
Having the imports and declarations like the following will fix the unit tests errors:
```
    imports: [HttpClientTestingModule, TabsModule.forRoot(), SharedModule],
    declarations: [
      OsdDetailsComponent,
      DeviceListComponent,
      SmartListComponent,
      TablePerformanceCounterComponent,
      OsdPerformanceHistogramComponent
    ],
```